### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by hoinmic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +29,6 @@ Version  | Codename  | LTS
 3.5      | Kirkstone | Yes
```
Comment (petermarko) [inline README.md]:
```suggestion
4.0      | Kirkstone | Yes
```

File: README.md [hunk 2]
```diff
@@ -27,7 +29,6 @@ Version  | Codename  | LTS
...
 3.4      | Honister  | No
 3.3      | Hardknott | No
-3.1      | Dunfell   | Yes
```
Comment (petermarko) [inline README.md:30]: There is still a working dunfell branch. Maybe reword "The current layer is compatible with the following Yocto version:" by stating that this table refers to master branch

Comment (hoinmic) [reply README.md:30]: Good hint.

File: README.md (hunk 1 above)
Comment (hoinmic) [reply README.md]: Thanks, I have modified it.
